dashboard/config/levels/custom/aichat/AI Chat Test Level.level

```diff
@@ -12,8 +12,42 @@
     "bot_title": "deprecated",
     "bot_description": "deprecated",
     "disable_projects": true,
+    "initial_ai_customizations": {
+      "botName": {
+        "value": "Ada Lovelace",
+        "visibility": "editable"
+      },
+      "temperature": {
+        "value": 0.5,
+        "visibility": "editable"
+      },
+      "systemPrompt": {
+        "value": "",
+        "visibility": "editable"
+      },
+      "retrievalContexts": {
+        "value": [
+
+        ],
+        "visibility": "editable"
+      },
+      "modelCardInfo": {
+        "value": {
+          "description": "",
+          "intendedUse": "",
+          "limitationsAndWarnings": "",
+          "testingAndEvaluation": "",
+          "exampleTopics": [
+
+          ]
+        },
+        "visibility": "editable"
+      },
+      "hidePresentationPanel": false
+    },
     "aichat_settings": {
       "initialCustomizations": {
+        "selectedModelId": "minstral",
```

Comment:
Spelling is 'mistral' [ref](https://huggingface.co/docs/transformers/main/en/model_doc/mistral).

Reply:
OOPS thanks for catching that!!